spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/AnnotatedServiceRegistrationBean.java

```diff
@@ -157,4 +170,32 @@ public AnnotatedServiceRegistrationBean setResponseConverters(
             ResponseConverterFunction... responseConverters) {
         return setResponseConverters(ImmutableList.copyOf(responseConverters));
     }
+
+    /**
+     * Returns sample requests of {@link #getService()}.
+     */
+    @NotNull
+    public Collection<AnnotatedExampleRequest> getExampleRequests() {
+        return exampleRequests;
+    }
+
+    /**
+     * Sets sample requests for {@link #getService()}.
+     */
+    public AnnotatedServiceRegistrationBean setExampleRequests(
+            @NotNull Collection<AnnotatedExampleRequest> exampleRequests) {
+        this.exampleRequests = exampleRequests;
```

Comment:
How about adding `exampleRequests` to the end of the existing field as we do in `HttpHeaders`?

Reply:
Sorry, I don't understand your comment. 😢 Could you explain more detail?

Reply:
Had a chat with @heowc. 😄

Reply:
Thank you for your help at night, @minwoox!!

In summary, 
There is generally a consistency that `setX` is adds to the end of the existing field. However, they are not classes like `~~RegistrationBean`. So these classes will go to [another PR](https://github.com/line/armeria/pull/2026#pullrequestreview-281918050) that needs refactoring.